de.fu_berlin.inf.dpp.whiteboard/frontend/src/components/menus.js

```diff
@@ -0,0 +1,100 @@
+import LOGIC from '../constants/logic.js';
+
+import drawRectImg from '../../assets/images/draw_rect.png';
```

Comment:
You should make a bundle and install that. You will be surprised that nothing works.

I guess that the contents will be bundled a litte bit different so I guess the src folder will not be present in the final bundle artifact.

Reply:
I did not fully understand what you meant with making a bundle first and then installing it, could please explain a little bit?

the way that it is currently done is that these imported images will be converted to [Base 64 URI](https://en.wikipedia.org/wiki/Data_URI_scheme#HTMLl) so that the actual image is embedded in the code and does not need to be fetched from anywhere.

[Live example](https://jsbin.com/yasojocuca/edit?js,output)

Reply:
yes I delayed uploading the build scripts because I have no idea how to setup and work with travis.
uploading the code first was of higher priority.

Reply:
@m273d15 can you please assist @AbdBarho in setting up the required stuff. In my opinion a running CI environment is more important than pushing code into the repo without verification.

Reply:
@AbdBarho Sorry, I missed this comment. What do you want to integrate into Travis?